.skiff/webapp.jsonnet

```diff
@@ -147,6 +147,12 @@ function(
             }
         },
         spec: {
+            strategy: {
+                type: 'RollingUpdate',
```

Comment:
Is the load balancer stateless? Does this mean during a deploy, two calls to the same API could hit different servers, and therefore different versions? Maybe that doesn't matter for these types of apps.

Reply:
Yup, there's no session stickiness or anything like that.

For our workloads the trade of makes sense. In cases where it matters (say Beaker, LB or the like) we just make sure to maintain backwards compatibility.